app/examples/vivaldi

```diff
@@ -0,0 +1,83 @@
+" Options
+set adblocker=update
+set cache=full
+set devtoolsposition=vsplit
+set favicons=30day
+set mintabwidth=0
+set permissioncamera=ask
+set permissiondisplaycapture=ask
+set permissionmediadevices=allow
+set permissionmicrophone=ask
+set permissionpersistentstorage=allow
+set redirecttohttp
+set search=https://google.com/search?q=%s
+set splitright
+set suspendonrestore=none
+set tabreopenposition=previous
+set windowtitle=title
+
+" Search Engines
+set search=https://www.bing.com/search?q=%s
```

Comment:
Right above you set the search to google, which of the two should it be?

Reply:
It is bing by default. This is fixed in the latest commits.